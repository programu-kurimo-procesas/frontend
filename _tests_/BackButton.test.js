import React from 'react'
import renderer from 'react-test-renderer'
import BackButton from '../src/components/BackButton'

describe('<BackButton />', () => {
    it('renders correctly', () => {
        const tree = renderer.create(<BackButton />).toJSON()
        expect(tree).toMatchSnapshot()
    })
    }
)