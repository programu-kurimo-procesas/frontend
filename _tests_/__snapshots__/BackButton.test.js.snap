// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<BackButton /> renders correctly 1`] = `
<View
  accessibilityState={
    {
      "busy": undefined,
      "checked": undefined,
      "disabled": undefined,
      "expanded": undefined,
      "selected": undefined,
    }
  }
  accessibilityValue={
    {
      "max": undefined,
      "min": undefined,
      "now": undefined,
      "text": undefined,
    }
  }
  accessible={true}
  collapsable={false}
  focusable={false}
  onClick={[Function]}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  style={
    {
      "left": 4,
      "opacity": 1,
      "position": "absolute",
      "top": 30,
    }
  }
>
  <Image
    source={1}
    style={
      {
        "height": 24,
        "width": 24,
      }
    }
  />
</View>
`;
